src/api/app/controllers/webui/package_controller.rb

```diff
@@ -81,6 +81,7 @@ def show
 
     @comments = @package.comments
     @requests = []
+    @services = Service.find(project: @project.name, package: @package.name)
```

Comment:
Because Service is not a subclass of `ApplicationRecord`. It is a subclass of `ActiveXML::Node` and the `exists?` method doesn't exist for that class.